### DISCUSSION TITLE
How to add a dag run with an old execution date

### DISCUSSION THREAD

**Discussion by RenGeng** [apache/airflow#24959] [Q&A]
On version 2.2.4 of airflow the icon to add a dag run on the `dagrun/list` endpoint disappeared, so is there any solution to create a dag run mannually ? I tried to backfill, but task are stuck in scheduled (cf. https://github.com/apache/airflow/issues/23145)

**Comment by potiuk** [comment]
I don't recall any "disappearance" - but what I generally can recommend is to upgrade - ideally to latest released `2.3.3` or if you want to be extremely cautious - to `2.2.5` which contains the latest bugfixes for 2.2 line.

**Reply by RenGeng** [reply]
For example, in the older version (2.1.1 in the screenshoot), there is a + button where we can manually add a dag run 
![image](https://user-images.githubusercontent.com/23097260/178263614-6d054e5b-32f0-4a31-88ab-2efb3800fd52.png)

and then we can specifie parameters to the dag run 
![image](https://user-images.githubusercontent.com/23097260/178263739-b2aa2fe3-823b-4e8f-bfdc-dc4db4d41388.png)

**Reply by potiuk** [reply]
I see. I think nothing was removed - you simply miss the right permission. Look through permissions and see if you find one that is missing (but I am not 100% sure - maybe it has been indeed simply removed, I find it slightly awkward to be able to add dag runs manually).
